src/Indicator.vala

```diff
@@ -200,9 +208,16 @@ public class Power.Indicator : Wingpanel.Indicator {
 
     private bool show_notification () {
         if (is_in_session) {
-            var notification = new Notify.Notification ("indicator-power", "", "display-brightness-symbolic");
-            notification.set_hint ("x-canonical-private-synchronous", new Variant.string ("indicator-power"));
-            notification.set_hint ("value", new Variant.int32 (dm.brightness));
+            Notify.Notification notification;
+            if (is_desktop) {
+                notification = new Notify.Notification ("indicator-power", "", "application-exit");
+                notification.set_hint ("x-canonical-private-synchronous", new Variant.string ("indicator-power"));
```

Comment:
I'm not familiar with UX, but what about just showing notification only when `is_desktop` is false, and don't show anything when `is_desktop`? Users should understand that nothing changed because nothing happens.

Even if we show the notification also when `is_desktop` is true, we should use another icon at least; the appearance of `application-exit` may be fine but as you can see from its name I guess it's not for this kind of use